### DISCUSSION TITLE
62

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#798] [Comments]
http://jojoldu.tistory.com/62

**Comment by jojoldu** [comment]
작성자: 지나가는프로그래머

수고하셨습니다. 잘보았습니다.

**Comment by baeharam** [comment]
창천향로님, 항상 잘 보고 있습니다. 매일 블로그 눈팅만 하다가 객체지향에 대해 좀더 알고싶어 포스트를 따라하면서 공부하는 중입니다. 제가 딱히 관심있는 분야가 없어서 고민하다가 존경하는 창천향로의 진로를 따라가고 싶다는 생각을 했습니다. 그래서 백엔드 개발자가 되기로 결정했는데 혹시 독학이 가능할까요? 현재 대학교 3학년에 재학중에 있습니다. 저도 코드스쿼드 같은 유료 학원을 다니면서 공부하고 싶지만 여건상 힘든 부분이 있네요... 대답해주시면 감사하겠습니다.

**Comment by deokju** [comment]
창천향로님 정말 잘 보았습니다.
저도 지금 따라하고 있는데.. 느므느므 어렵네요.
객체간의 역할이라.. 이 말이 참 어렵네요 정말
